src/main/resources/db/migration/V43_45__Create_human_detected_object_table.sql

```diff
@@ -0,0 +1,7 @@
+create table if not exists "human_detected_object" (
```

Comment:
Why can't we use the same object table here ? What are the attributes that doesn't match between this new `human_detected_object` and the "machine  detected object"

Reply:
It is because there is an object named Label (id, name, color) from Annotator which can not be mapped to DetectableObjectType